### PR TITLE
Added the stay function and a user test

### PR DESCRIPTION
**Pull request by athai**

### CODE REVIEW

File: src/main/java/models/Game.java [hunk 1]
```diff
@@ -87,6 +87,20 @@ else if (dealerScore != userScore)
 			userWon = true;
 	}
 
+    // A player choosing to stay is essentially a signal
+    // for the dealer's turn to start
+    public void stay(){
+        dealCardToDealer(); // Call twice at turn start
+        dealCardToDealer();
+        // Do we need to turn one of these cards face down? Or will that
```
Comment (leel8on) [inline src/main/java/models/Game.java]: This actually needs to be done at the start of the game. The user gets to see what one of the two cards the dealer has while they are playing. I can just do that from the deal function in the controller I think.

Comment (athai) [inline src/main/java/models/Game.java]: All right. I know the card class has two boolean functions that will set whether it should be face up or not. Maybe you can use those?

Comment (leel8on) [inline src/main/java/models/Game.java]: possible, though I have another idea

**Comment by leel8on** [comment]
:+1:

File: src/main/java/models/Game.java [hunk 2]
```diff
@@ -77,16 +77,33 @@ public void dealCardToDealer()
...
+            user.loseBet(ante);
+        }else if (dealerScore != userScore){
+            userWon = true;
+            user.winBet(ante);
+        }
 	}
 
+    // A player choosing to stay is essentially a signal
+    // for the dealer's turn to start
+    public void stay(){
```
Comment (IStallcup) [inline src/main/java/models/Game.java:95]: Brilliant! :fireworks:

**Comment by IStallcup** [comment]
Looks great!

File: src/main/java/models/Game.java [hunk 3]
```diff
@@ -77,16 +77,33 @@ public void dealCardToDealer()
...
-		else if (dealerScore != userScore)
-			userWon = true;
+	public void whoWon(int ante)
+    {
+        int dealerScore = dealer.calcScore(dealer.hand);
+        int userScore = user.calcScore(user.hand);
+        if (dealerScore > userScore){
+            dealerWon = true;
+            user.loseBet(ante);
+        }else if (dealerScore != userScore){
```
Comment (kannabanana) [inline src/main/java/models/Game.java:87]: Do we have the new game function to "deallocate" the array and remove the users hands? But I guess that would be the empty array function, but does it actually connect?

Comment (athai) [inline src/main/java/models/Game.java:87]: There's a separate function further down in the code called newTurn() that will reset the boolean values and empty the user and dealer hands. :)

**Pull request merged**